caffe2/contrib/tensorrt/tensorrt_tranformer.cc

```diff
@@ -455,20 +413,69 @@ void TensorRTTransformer::ClusterToTrtOp(
   model->mutable_graph()->clear_initializer();
 }
 
+CaffeMap<std::string, TensorShape> TensorRTTransformer::SsaRewriteAndMapIO(
```

Comment:
What does IO here mean?
Looks to me what this function does is rewrite pred_net to ssa form and adjust the shape hints with the new mapping.

Reply:
Yeah, the shape hints are mostly about input shapes. But I agree that the name is not accurate. Will change.